bazelrunner/src/main/java/org/jetbrains/bsp/bazel/bazelrunner/BazelRunner.java

```diff
@@ -19,9 +23,27 @@
 
   private Optional<Integer> besBackendPort = Optional.empty();
   private Optional<BuildClientLogger> buildClientLogger = Optional.empty();
+  private final Supplier<File> workspaceRoot;
 
   public BazelRunner(String bazelBinaryPath) {
     this.bazel = bazelBinaryPath;
+    this.workspaceRoot = Suppliers.memoize(this::resolveWorkspaceRoot);
+  }
+
+  private File resolveWorkspaceRoot() {
+    var builder = new ProcessBuilder(bazel, "info", "workspace");
+    try {
+      var process = new BazelProcess(builder.start(), Optional.empty());
+      var result = process.waitAndGetResult();
+      if (result.getStatusCode() != StatusCode.OK) {
+        throw new RuntimeException(
+            "Failed to run bazel info workspace. Make sure that the project is created inside a"
+                + " bazel workspace");
+      }
```

Comment:
```
bazelRunner
    .commandBuilder()
    .info()
    .withArgument(argument)
    .executeBazelCommand()
    .waitAndGetResult();
```

Reply:
But this is the bazel runner that I am changing :p I want to make sure that bazel is run from workspace root to avoid problems with potential relative paths to targets. To achieve this, bazel runner must create process with cwd set to workspace root. I need to first resolve what this root is. I can't use bazel runner for it as it is not yet ready because it doesn't have this information.

Reply:
ohh right, then it sucks a bit (i mean bazel runner, not your fault), okey so we can merge it